### ISSUE TITLE
No space after class name generates: Class name "" is not in camel caps format

### ISSUE THREAD

**Issue by lucascube** [squizlabs/PHP_CodeSniffer#663]
So I'm new to CodeSniffer and I was hoping to get a good understanding as to why this error keep occurring on existing and new code. The error is Class name "" is not in camel caps format and it happens using PSR1.
```
phpcs --standard=PSR1 '/var/www/webroot/paypalipn/Test/CodeSniffer/Test.php'

FILE: /private/var/www/webroot/paypalipn/Test/CodeSniffer/Test.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 15 | ERROR | Class name "" is not in camel caps format
----------------------------------------------------------------------

Time: 9ms; Memory: 2.5Mb
```

FILE CONTENT:
```php
<?php
/**
 * File DOC
 */
namespace Test\CodeSniffer;

/**
 * Class Test
 *
 * @package Test\CodeSniffer
 * @author  ${USER} <${USER}@datajoe.com>
 * @date    ${DATE}
 * @version 1
 */
class Test{
	/**
	 * constructor
	 */
	public function __construct(){
		echo "Testing";
	}
}
```
```
phpcbf --standard=PSR1 --encoding=utf-8 -w -v -a -s '/var/www/webroot/paypalipn/Test/CodeSniffer/Test.php'
Changing into directory /private/var/www/webroot/paypalipn/Test/CodeSniffer
Processing Test.php [PHP => 97 tokens in 22 lines]... DONE in 8ms (0 fixable violations)
No fixable errors were found
Time: 12ms; Memory: 2.25Mb
```
A more verbose run:
```
phpcs --standard=PSR1 --encoding=utf-8 -w -v -a -s '/var/www/webroot/paypalipn/Test/CodeSniffer/Test.php'
Registering sniffs in the PSR1 standard... DONE (7 sniffs registered)
Creating file list... DONE (1 files in queue)
Changing into directory /private/var/www/webroot/paypalipn/Test/CodeSniffer
Processing Test.php [PHP => 97 tokens in 22 lines]... DONE in 5ms (1 errors, 0 warnings)

FILE: /private/var/www/webroot/paypalipn/Test/CodeSniffer/Test.php
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 15 | ERROR | Class name "" is not in camel caps format
    |       | (Squiz.Classes.ValidClassName.NotCamelCaps)
----------------------------------------------------------------------
```

**Comment by aik099** [comment]
Please do this to made bug report more readable:

* format `phpcs` command output using fenced code blocks
* put file contents in question in a gist or format it using fenced code blocks as well

**Comment by gsherwood** [comment]
I've fixed the comment.

**Comment by gsherwood** [comment]
This was caused by a specific sniff assuming there would be a space between the class name and the opening brace, which is obviously not the case in your code. I've now fixed this bug.

**Comment by lucascube** [comment]
Thank you for fixing my post. I'll try to remember to format it in future use. Also, another thank you for fixing this bug.
